org.opent2t.onboarding.getverahub/js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "opent2t-onboarding-org-opent2t-onboarding-getverahub",
```

Comment:
unclear, the pattern we're following is com-hubsite, for vera its getvera.com. 

---
In reply to: [103515875](https://github.com/openT2T/onboarding/pull/32#discussion_r103515875) [](ancestors = 103515875)